modules/compliance-rescan.adoc

```diff
@@ -10,3 +10,8 @@ Typically you will want to re-run a scan on a defined schedule, like every Monda
 ----
 $ oc annotate compliancescans/<scan_name> compliance.openshift.io/rescan=
 ----
+
+[NOTE]
+====
+When the scan setting `default-auto-apply` label is applied, remediations are applied automatically and outdated remediations will also automatically update. If there are remediations that have not been applied due to dependencies, or remediations that had been outdated, re-scanning will apply the remediations and might trigger a reboot. Only remediations that use `MachineConfig` objects trigger reboots. If there are no updates or dependencies to be applied, no reboot occurs.
```

Comment:
Looks like elsewhere in the module it's not hyphenated, so: s/re-scanning/rescanning/ ?

Reply:
There's a couple "will"s that might be able to be dropped or rewritten to avoid future tense, if that still makes sense in context.

Reply:
Wondering now if `[NOTE]` should get upgraded to `[IMPORTANT]` or `[WARNING]` since it involves a potential reboot?